build/linter/constructor/analyzer_test.go

```diff
@@ -28,6 +28,6 @@ import (
 
 func Test(t *testing.T) {
 	testdata := analysistest.TestData()
-	pkgs := []string{"t", "github.com/pingcap/tidb/util/linter/constructor"}
+	pkgs := []string{"t", "github.com/pingcap/tidb/pkg/util/linter/constructor"}
```

Comment:
maybe declare a const for the package prefix to reduce duplication. And maybe use some runtime reflections to ensure it's updated after the folder is changed.

Reply:
Done.